src/inventory.h

```diff
@@ -271,7 +281,7 @@ class InventoryList
 private:
 	std::vector<ItemStack> m_items;
 	std::string m_name;
-	u32 m_size;
+	u32 m_size; // always the same as m_items.size()
```

Comment:
Did you keep this variable intentionally for a future cleanup PR, or is it somehow still needed?

Reply:
I just didn't want to touch even more code and haven't looked if this variable is very useful.